sct.py

```diff
@@ -45,6 +47,14 @@ def install_callback(ctx, _, value):
     return sys.exit(0)
 
 
+def add_file_logger(level=logging.DEBUG):
+    cmd_path = "-".join(click.get_current_context().command_path.split()[1:])
+    filename = datetime.datetime.now().strftime(f"hydra-{cmd_path}-%Y%m%d-%H%M%S.log")
+    handler = logging.FileHandler(os.path.join(Setup.logdir(), filename))
```

Comment:
this might be a bit problematic for some of the stage, since Setup.logdir() might be on the code path to change the `latest` dir, and there are is logic based on that, i.e. looking to the test_id file.

need run this in a job, to see we are not shooting our legs here...

Reply:
Good point. Reworked it to create separate directory.